tests/xcconfig/BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+load("//rules:framework.bzl", "apple_framework")
+load(":tests.bzl", "xcconfig_unit_test_suite")
+
+SRCS = [
+    "empty.m",
+    "empty.swift",
+]
+
+EXPECTED_FAILURE = ["manual"]
+
+apple_framework(
+    name = "empty_xcconfig",
+    srcs = SRCS,
+    xcconfig = {},
+)
+
+apple_framework(
+    name = "unknown_key_xcconfig",
+    srcs = SRCS,
+    xcconfig = {
+        "MY_MADEUP_SETTING": "NEVER",
+    },
+)
+
+apple_framework(
+    name = "boolean_key_xcconfig",
+    srcs = SRCS,
+    xcconfig = {
+        "GCC_TREAT_WARNINGS_AS_ERRORS": "YES",
+    },
+)
+
+# apple_framework(
```

Comment:
Assume this is a case of "uncomment to make sure that this fails as expected"? Might be worth having a comment explaining the commented code.

Reply:
Bump

Reply:
This diff is outdated, commented-out code has been removed